x/launch/types/vesting_options.go

```diff
@@ -1,4 +1,23 @@
 package types
 
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
 // VestingOptions defines the interface for vesting options
-type VestingOptions interface{}
+type VestingOptions interface {
+	Validate() error
+}
+
+var _ VestingOptions = &DelayedVesting{}
+
+// Validate check the DelayedVesting object
+func (g DelayedVesting) Validate() error {
+	if g.Vesting.Empty() || !g.Vesting.IsValid() {
+		return sdkerrors.Wrap(ErrInvalidCoins, "invalid vesting coins for DelayedVesting")
+	}
+	if g.EndTime == 0 {
```

Comment:
Although we can have a empty `StartingBalance` we should still verify the the value provided is valid.

So we can check `IsValid()` without `Empty()`